crates/credentials/src/verifiable_credential.rs

```diff
@@ -130,6 +130,52 @@ pub struct VerifiableCredential {
     pub credential_subject: CredentialSubject,
 }
 
+impl VerifiableCredential {
+    fn into_jwt_payload_verifiable_credential(self) -> JwtPayloadVerifiableCredential {
+        JwtPayloadVerifiableCredential {
+            context: self.context,
+            id: Some(self.id),
+            r#type: self.r#type,
+            issuer: Some(self.issuer),
+            issuance_date: Some(self.issuance_date),
+            expiration_date: self.expiration_date,
+            credential_subject: Some(self.credential_subject),
+        }
+    }
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+struct JwtPayloadVerifiableCredential {
+    #[serde(rename = "@context")]
+    context: Vec<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    id: Option<String>,
+    #[serde(rename = "type")]
+    r#type: Vec<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    issuer: Option<Issuer>,
+    #[serde(skip_serializing_if = "Option::is_none", rename = "issuanceDate")]
+    issuance_date: Option<i64>,
+    #[serde(skip_serializing_if = "Option::is_none", rename = "expirationDate")]
+    expiration_date: Option<i64>,
+    #[serde(skip_serializing_if = "Option::is_none", rename = "credentialSubject")]
+    credential_subject: Option<CredentialSubject>,
+}
+
+impl JwtPayloadVerifiableCredential {
+    fn into_verifiable_credential(self) -> VerifiableCredential {
+        VerifiableCredential {
+            context: self.context,
+            id: self.id.unwrap_or_default(),
+            r#type: self.r#type,
+            issuer: self.issuer.unwrap(),
```

Comment:
`unwrap()` is scary. that's a , i.e. the program crashes. we should probably return a `Result` instead

Reply:
no more unwraps 👍